### DISCUSSION TITLE
Please  fix the orchid repeater field or add another solution.

### DISCUSSION THREAD

**Discussion by estudiooka** [orchidsoftware/platform#1935] [Ideas]
Orchid is amazing, problably the best solution out there , even better than the paid ones.

But the lack of a repeater  field make it unsuable for the modern CMS style.

Please make it a priority the orchid-repeater-field is not working any more. 

I don't want to buy nova only because the repeater field.

I can help if need. 

A repeater field is very very important.

**Comment by tabuna** [comment]
We are an open-source project. You can participate and contribute or release your extension.
